diff --git a/package/libcurl/Config.in b/package/libcurl/Config.in
index d8072c0afc..d6cbcabaa6 100644
--- a/package/libcurl/Config.in
+++ b/package/libcurl/Config.in
@@ -1,5 +1,6 @@
 config BR2_PACKAGE_LIBCURL
 	bool "libcurl"
+	default y
 	help
 	  cURL is a tool for getting files from FTP, HTTP, Gopher,
 	  Telnet, and Dict servers, using any of the supported
@@ -11,11 +12,13 @@ if BR2_PACKAGE_LIBCURL
 
 config BR2_PACKAGE_LIBCURL_CURL
 	bool "curl binary"
+	default y
 	help
 	  Install curl binary as well
 
 config BR2_PACKAGE_LIBCURL_VERBOSE
 	bool "verbose strings"
+	default y
 	help
 	  Enable verbose text strings
 
@@ -50,27 +53,33 @@ choice
 
 config BR2_PACKAGE_LIBCURL_OPENSSL
 	bool "OpenSSL"
+	default y
 	depends on BR2_PACKAGE_OPENSSL
 	select BR2_PACKAGE_LIBOPENSSL_ENABLE_DES if BR2_PACKAGE_LIBOPENSSL
 
 config BR2_PACKAGE_LIBCURL_BEARSSL
 	bool "BearSSL"
+	default y
 	depends on BR2_PACKAGE_BEARSSL
 
 config BR2_PACKAGE_LIBCURL_GNUTLS
 	bool "GnuTLS"
+	default y
 	depends on BR2_PACKAGE_GNUTLS
 
 config BR2_PACKAGE_LIBCURL_LIBNSS
 	bool "NSS"
+	default y
 	depends on BR2_PACKAGE_LIBNSS
 
 config BR2_PACKAGE_LIBCURL_MBEDTLS
 	bool "mbed TLS"
+	default y
 	depends on BR2_PACKAGE_MBEDTLS
 
 config BR2_PACKAGE_LIBCURL_WOLFSSL
 	bool "WolfSSL"
+	default y
 	depends on BR2_PACKAGE_WOLFSSL
 	select BR2_PACKAGE_WOLFSSL_ALL
 
